src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+body {
+  margin: 0;
+  text-transform: uppercase;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
+.navigation__bar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: rgb(255 255 255);
+  box-shadow: 0 2px 4px rgb(0 0 0 / 5%);
+}
+
+.nav_list {
+  display: flex;
+  padding: 0;
+  margin: 0;
```

Comment:
You can remove this padding and margin from this selector as your reset.css will do the job of reseting padding and margins:
```suggestion

```